From 5ccfe1565a8039f48b0ca4de0849fcd8fe7a1ec5 Mon Sep 17 00:00:00 2001
From: "Thulasimani,Sivakumar" <sivakumar.thulasimani@intel.com>
Date: Thu, 31 Mar 2016 23:29:11 +0530
Subject: [PATCH 1/9] Restrict short pulse delay to 1ms

waiting for a full 2ms to generate short pulse results in
display driver getting interrupt as long pulse. Hence this
patch reduces the delay for short pulse to 1ms which
results in display driver getting the interupt as short
pulse.

Change-Id: I78e247e96e4a0d242652f4a163a6622914c2769b
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-27637
Signed-off-by: Sivakumar Thulasimani <sivakumar.thulasimani@intel.com>
Reviewed-on: https://android.intel.com:443/489925
---
 drivers/usb/typec/pd/vdm_process.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/pd/vdm_process.c b/drivers/usb/typec/pd/vdm_process.c
index 21f3372..5e73e50 100644
--- a/drivers/usb/typec/pd/vdm_process.c
+++ b/drivers/usb/typec/pd/vdm_process.c
@@ -56,7 +56,7 @@
 #define DISP_PORT_SIGNAL_GEN2		2
 
 #define DP_HPD_LONG_PULSE_TIME		5 /* 5 mSec */
-#define DP_HPD_SHORT_PULSE_TIME		2   /* 2 mSec */
+#define DP_HPD_SHORT_PULSE_TIME		1   /* 1 mSec */
 #define DP_HPD_AUTO_TRIGGER_TIME	200 /* 200 mSec */
 
 static void pe_prepare_vdm_header(struct vdm_header *v_hdr, enum vdm_cmd cmd,
-- 
1.9.1

